### PR TITLE
[SDK-623] feat: update api to work with GLTFast 6.0

### PR DESCRIPTION
**Pull request by HarrisonHough**
## Description

-   This PR includes updates to support GLTF 6.0 it is a breaking change and it seems there isn't an elegant way to add or handle backwards compatibility. 

## How to Test

-  Importy gltfast 6.0 and our SDK (this branch) everything should work without errors

## Checklist

-   [ ] Tests written or updated for the changes.
-   [ ] Documentation is updated.
-   [ ] Changelog is updated.

[SDK-623]: https://ready-player-me.atlassian.net/browse/SDK-623?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Runtime/GLTF/GltFastGameObjectInstantiator.cs [hunk 1]
```diff
@@ -41,8 +40,6 @@ public class GltFastGameObjectInstantiator : GameObjectInstantiator
             {
                 // Use Node GameObject for first Primitive
                 meshGo = m_Nodes[nodeIndex];
-                // Ready Player Me - Parent mesh to Avatar root game object
-                meshGo.transform.SetParent(m_Parent.transform);
```
Comment (rk132) [inline Runtime/GLTF/GltFastGameObjectInstantiator.cs:45]: do you know, why was the parent changed to the root game object?

**Review by rk132 — APPROVED** [review]
Looks good. One small question.

File: Runtime/GLTF/GltFastGameObjectInstantiator.cs (hunk 1 above)
Comment (HarrisonHough) [reply Runtime/GLTF/GltFastGameObjectInstantiator.cs:45]: ah good point, let me check. Probably I need to add this back in

Comment (HarrisonHough) [reply Runtime/GLTF/GltFastGameObjectInstantiator.cs]: I added it back in. But when testing it out I couldn't notice any difference. Will check with robin once he is back

**Pull request merged**